server/utils/crmFieldFormatter.ts

```diff
@@ -67,3 +67,10 @@ export const formatPercentage = (value: string): string => {
 export const splitCamelCase = (value: string): string => {
```

Comment:
Can we move splitCamelCase & removeUnderscore to utils.ts and add test cases.
So this file only has formatXXX functions.